src/plugins/intel_cpu/src/nodes/pooling.cpp

```diff
@@ -146,13 +146,19 @@ dnnl::pooling_forward::primitive_desc createDescriptorHelper(const dnnl::engine&
 
 bool Pooling::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::string& errorMessage) noexcept {
     try {
-        if (ov::is_type<const ov::op::v8::MaxPool>(op)) {
+        if ((ov::is_type<const ov::op::v14::MaxPool>(op) || ov::is_type<const ov::op::v14::AvgPool>(op)) && op->is_dynamic()) {
+            errorMessage = "ov::op::RoundingType::CEIL_TORCH is not supported with dynamic shapes";
+            return false;
```

Comment:
The `v14::MaxPool` and `v14::AvgPool` can be created not only with `CEIL_TORCH` `ov::op::RoundingType`.
Currently ops with other rounding types are converted by downgrade transformation. But with dwongrade disabled other rounding types with dynamic shape will fail (or fallback to ref) with "CEIL_TORCH" in the message anyway.
To match current condition, It should be rephrased to "MaxPool-14 and AvgPool-14 are not supported with dynamic shapes". Or check the RoundingType as well if that's the restriction.

Reply:
Fixed, the code for v14 is now properly tested.